ci/pipelines/e2e-helpers.yml

```diff
@@ -468,40 +496,49 @@ jobs:
       put: slack-notification
       attempts: 10
       params:
-        channel: '#govuk-pay-starling'
+        channel: "#govuk-pay-starling"
         silent: true
-        text: ':red-circle: Failed to build and push e2e helper reverse-proxy - <https://pay-cd.deploy.payments.service.gov.uk/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|Concourse build #$BUILD_NAME>'
+        text: ":red-circle: Failed to build and push e2e helper reverse-proxy - <https://pay-cd.deploy.payments.service.gov.uk/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|Concourse build #$BUILD_NAME>"
         icon_emoji: ":concourse:"
         username: pay-concourse
     on_success:
       put: slack-notification
       attempts: 10
       params:
-        channel: '#govuk-pay-activity'
+        channel: "#govuk-pay-activity"
         silent: true
-        text: ':green-circle: Built and pushed e2e helper reverse-proxy - <https://pay-cd.deploy.payments.service.gov.uk/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|Concourse build #$BUILD_NAME>'
+        text: ":green-circle: Built and pushed e2e helper reverse-proxy - <https://pay-cd.deploy.payments.service.gov.uk/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|Concourse build #$BUILD_NAME>"
         icon_emoji: ":concourse:"
         username: pay-concourse
 
   - name: reverse-proxy-e2e
     plan:
       - in_parallel:
-        - get: reverse-proxy-candidate-ecr-registry-test 
-          passed: [build-and-push-reverse-proxy-candidate]
-          trigger: true
-          params:
-            format: oci
-        - get: pay-ci
+          - get: reverse-proxy-candidate-ecr-registry-test
+            passed: [build-and-push-reverse-proxy-candidate]
+            trigger: true
+            params:
+              format: oci
+          - get: pay-ci
+      - task: parse-candidate-tag
+        file: pay-ci/ci/tasks/parse-candidate-tag.yml
+        input_mapping:
+          ecr-repo: reverse-proxy-candidate-ecr-registry-test
+      - load_var: release_number_tag
+        file: tags/release-number
```

Comment:
This should be fine though - not sure why the pipeline is orange for `reverse-proxy-e2e` (unless this just needs another re-run?)